### ISSUE TITLE
Noise disposal methods

### ISSUE THREAD

**Issue by dadaban** [ZPGuiGroupWhu/ClusteringDirectionCentrality#7]
Hi, may I ask whether the first two noise disposal methods mentioned in Synthetic Data Analysis are original? If not, can you provide some references? 
Thanks.

**Comment by pdh-coder** [comment]
Yes, they can be considered original because they are based on very simple ideas, both relying on local density. One is achieved using RKNN, while the other is achieved through averaging KNN distances. Hope this can help you.
